infor-design/enterprise
# 4862 - Fix pager to use tabs instead arrow keys

**Explain the _details_ for making this change. What existing problem does the pull request solve?**
- Fixed accessibility issue to use tabs instead arrow keys with Pager.
- Fixed pagesize pull down to open with arrow down key
- For disapper tooltip will handle separate issue #5161 

**Related github/jira issue (required)**:
Closes #4862

**Steps necessary to review your pull request (required)**:
Pull this branch and build/run the demo app
- Navigate to: http://localhost:4000/components/pager/example-standalone.html
- Pager button should be able to navigate by tab key
- Click to activate the setting `Show Page Size Selector`
- Use keyword to navigate that pull down in pager bar
- See it should be able to open by down arrow key

Test NG
- Pull and build NG branch ([main](https://github.com/infor-design/enterprise-ng.git))
- Copy and replace "dist" folder from `enterprise` to `⁨enterprise-ng⁩/node_modules⁩/ids-enterprise`
- Navigate to http://localhost:4200/ids-enterprise-ng-demo/datagrid-standalone-pager
- Pager button should be able to navigate by tab key
- Use keyword to navigate pagesize pull down in pager bar
- See it should be able to open by down arrow key
**Included in this Pull Request**:
